tasks/review-slack-integrations.md

```diff
@@ -0,0 +1,17 @@
+# GCI Tasks: Comparison with Slack integrations
+
+## Background
+
+A Zulip integration is a special type of bot that brings information from the outside world into Zulip. Zulip integrations allow teams to stay up-to-date on code changes, issue tickets, build system results, tweets, and more. Zulip currently has over 50 integrations, but there are dozens more that would be valuabled to add.
+
+To get an idea of the type of integrations out there, we've grabbed a list of all the apps available for Slack, another chat tool. We'd like some analysis of thess applications, in particular some information about how the integration works (is it a webhook, a slash command, a command-line script or something else?) and whether it's something that would be good to implement for Zulip.
+
```

Comment:
Maybe we should add a link to Slack's API docs to explain what all these things are?